### PR TITLE
Add support for open-ended survey questions and add podcast topics survey

### PR DESCRIPTION
**Pull request by gabrielKerekes**
- a question can be marked as `is_open_ended`
- a text answer is expected for such questions
- the answer can be edited by editing the original message

I've tested the previous survey functionality. I've also tested a survey with mixed questions, multiple open-ended questions and everything seems to be working quite nicely. 

## TODO
- [x] Remove "Vitaj xxx" from survey intro message
- [x] Add questions from https://www.notion.so/Dotaznik-podcast-topics-50b615af87ab40f98046b2aa89321ec5

### CODE REVIEW

**Comment by xjahic** [comment]
I would like to add more questions to the survey. Use this opportunity (I think it's a little bit waste to send survey with just a one question). You can find my suggestions [https://www.notion.so/Dotaznik-podcast-topics-50b615af87ab40f98046b2aa89321ec5](here)

File: migrations/0005.podcast-topics-survey.sql [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+INSERT INTO survey_question(survey_id, _order, text, is_multiple_choice, is_open_ended)
+VALUES(3, 4, E'Akých hosťov (z akých odborov) by si chcel/a počuť v podcaste?', FALSE, TRUE);
+
+-- question id 18
+INSERT INTO survey_question(survey_id, _order, text, is_multiple_choice, is_open_ended)
+VALUES(3, 5, E'Máš niečo, k čomu by si chcel/a počuť náš názor? Je niečo, čo sme už dávno mali prebrať, ale ešte sme tak neurobili? Nerozumieš nejakej téme a myslíš si, že by ti mohla nejaká epizódka alebo video pomôcť?', FALSE, TRUE);
+
+-- question id 19
+INSERT INTO survey_question(survey_id, _order, text, is_multiple_choice)
+VALUES(3, 6, 'Si si istý/á svojou odpoveďou? Ked pridáš 👍, tak odpoveď už nebudeš môcť meniť a dokončíš dotazník.', FALSE);
```
Comment (xjahic) [inline migrations/0005.podcast-topics-survey.sql:54]: Zmenil by som na "Si si isty/a svojimi odpovedami? ... odpovede uz nebudes..
resp mnozne cislo namiesto jednotneho

**Comment by xjahic** [comment]
Everything is fine except last question (check comment) and one more thing.
I tested it and changing the single-choice answer doesn't work anymore. Error: ` ERROR: Error while adding survey answer. Both survey_answer_id and survey_answer_text are None`
I tried to change answer in previous survey and it doesn't work there as well. I  don't know if the mistake was already there or whether you somehow caused it. Can you check that and maybe fix it? Or one of us can fix in another MR

**Review by xjahic — CHANGES_REQUESTED** [review]
Everything is fine except last question (check comment) and one more thing.
I tested it and changing the single-choice answer doesn't work anymore. Error: `ERROR: Error while adding survey answer. Both survey_answer_id and survey_answer_text are None`
I tried to change answer in previous survey and it doesn't work there as well. I don't know if the mistake was already there or whether you somehow caused it. Can you check that and maybe fix it? Or one of us can fix in another MR

File: migrations/0005.podcast-topics-survey.sql (hunk 1 above)
Comment (gabrielKerekes) [reply migrations/0005.podcast-topics-survey.sql]: Updated in https://github.com/StreetOfCode/discord-bot/pull/37/commits/cd065f2a51092859d642fb8918338a05299016cd. I decided to use the same wording as we use in feedback survey.

**Comment by gabrielKerekes** [comment]
> Can you check that and maybe fix it?

Nice catch. It was a bug I caused. I don't know how it got past QA though - I will check with them. This should definitely **not** happen. ... Fixed in https://github.com/StreetOfCode/discord-bot/pull/37/commits/2e10205bc27edc736f073b9dcf36f58fa42bba69.

**Review by xjahic — APPROVED** [review]
Great you fixed the issue. QA are not doing their job well. Hmmm. They definitely don't deserve Christmass bonus

**Pull request merged**